styleguide/components/31-molecules/menu/_menu.scss

```diff
@@ -42,19 +89,23 @@ nav.menu {
     height: 100%;
     background: rgba(255, 255, 2555, .8);
     z-index: 9998;
+
+    @include tablet {
+      display: none;
+    }
   }
 
   button {
-    @include button-small;
```

Comment:
Why not use a predefined button mixin anymore?

Reply:
Font-size etc are very specific for this molecule, so actually including would be overhead.